integration/token/fungible/views/transfer.go

```diff
@@ -24,6 +25,8 @@ import (
 
 // Transfer contains the input information for a transfer
 type Transfer struct {
+	// Auditor is the name of the auditor that must be contacted to approve the operation
```

Comment:
name vs enrolment ID?

Reply:
not the EID but just the name/label used in the configuration to identify the FSC auditor identity